validator/server_arb/execution_run.go

```diff
@@ -105,21 +105,10 @@ func (e *executionRun) machineHashesWithStepSize(
 	if err != nil {
 		return nil, err
 	}
-	log.Debug(fmt.Sprintf("Advanced machine to index %d, beginning hash computation", machineStartIndex))
-
-	// In BOLD, the hash of a machine at index 0 is a special hash that is computed as the
-	// `machineFinishedHash(gs)` where `gs` is the global state of the machine at index 0.
-	// This is so that the hash aligns with the start state of the claimed challenge edge
-	// at the level above, as required by the BOLD protocol.
-	var machineHashes []common.Hash
-	if machineStartIndex == 0 {
-		gs := machine.GetGlobalState()
-		log.Debug(fmt.Sprintf("Start global state for machine index 0: %+v", gs))
-		machineHashes = append(machineHashes, machineFinishedHash(gs))
-	} else {
-		// Otherwise, we simply append the machine hash at the specified start index.
-		machineHashes = append(machineHashes, machine.Hash())
-	}
+	log.Info(fmt.Sprintf("Advanced machine to index %d, beginning hash computation", machineStartIndex))
```

Comment:
I'd clarify this log line a bit, maybe "Advanced WASM machine to index %d, beginning challenge hash computation" or something to clarify this is about a challenge. Also, machineStartIndex should be a log parameter not part of the log message itself